options/locale/locale_en-US.ini

```
@@ -1007,6 +1008,8 @@ delete_preexisting_success = Deleted unadopted files in %s
 blame_prior = View blame prior to this change
 author_search_tooltip = Shows a maximum of 30 users
 
+tree_not_found = The %s branch of %s does not contain the path %s
```

```suggestion
tree_path_not_found = Path %[2]s doesn't exist in branch %[2]s
```

Overall LGTM, while I think it could be like this.

* Use `%[1]s` to make it easier to translate (move the position).
* The repo name is already on the page, so do not need to mention it again.